### PR TITLE
WingetBot but poggers

### PR DESCRIPTION
**Pull request by Yomanz**
Changes from non-poggers widgetbot to officially certified poggers widgetbot
this pr is less than untested.

those values in settings.json need changing to one of these so called "poggers" instances of widgetbot, don't commit them here tho!
i don't want anyone having my poggers instance information
advaith has poggers instance info he can put into production

thank,s
WingetBot taem;

### CODE REVIEW

**Comment by victiondev** [comment]
sex

**Comment by Daxorinator** [comment]
stonks! +1 for wigetbut

**Comment by Yomanz** [comment]
> 
> 
> stonks! +1 for wigetbut

@advaith1 please remove this spam, thanks

**Comment by victiondev** [comment]
As a member of the Poggers Society of England, Yomanz has personally received a letter of certification for this Push Request.

File: src/Routes/bots.ts [hunk 1]
```diff
@@ -292,9 +292,13 @@ router.post(
                     });
 
             if (fetchServer)
-                await fetch(
-                    `https://stonks.widgetbot.io/api/graphql?query={guild(id:"${req.body.widgetServer}"){id}}`
-                ).then(async (fetchRes: fetchRes) => {
+            await fetch(`https://${settings.widgetbot.backend}/api/graphql`, {
+                method: 'post',
+                body:    JSON.stringify({
```
Comment (advaith1) [inline src/Routes/bots.ts:297]: in all instances of this, there should be a single space after `body:`

File: assets/Views/templates/docs/index.ejs [hunk 2]
```diff
@@ -119,7 +119,7 @@
         const crate = new Crate({
           server: "<%= settings.guild.main %>",
           channel: "<%= settings.channels.docsDefault %>",
-          shard: 'https://e.widgetbot.io'
+          shard: '<%= settings.widgetbot.frontend %>'
```
Comment (advaith1) [inline assets/Views/templates/docs/index.ejs:122]: docs should use v3 since it's the del server

**Review by advaith1 — CHANGES_REQUESTED** [review]
instead of `settings.widgetbot.frontend` (for v2) would be better to just omit `shard` because that is now the default in the client libs (also would be more consistent; looks like shard is already not present in some areas)

also is it necessary to put the v2 backend url in settings? it's public so it doesn't seem necessary

**Review by advaith1 — APPROVED** [review]

**Pull request merged**